src/test/java/io/quarkus/registry/app/services/MavenResourceTest.java

```diff
@@ -62,7 +86,32 @@ void should_return_platforms() {
         given()
                 .get("/maven/io/quarkus/registry/quarkus-platforms/1.0-SNAPSHOT/quarkus-platforms-1.0-SNAPSHOT.json")
                 .then()
-                .statusCode(200);
+                .statusCode(200)
+                .log().ifValidationFails()
+                .contentType(ContentType.JSON)
+                .body("platforms.streams.flatten().size()", is(3),
+                        "platforms[0].streams[0].id", is("2.0"),
+                        "platforms[0].streams[0].releases[0].quarkus-core-version", is("2.0.2.Final"),
+                        "platforms[0].streams[1].id", is("1.9"),
+                        "platforms[0].streams[1].releases[0].quarkus-core-version", is("1.9.0.Final"),
+                        "platforms[0].streams[2].id", is("2.1"),
+                        "platforms[0].streams[2].releases[0].quarkus-core-version", is("2.1.0.CR1"));
+    }
+
+    @Test
+    void should_return_all_platforms() {
+        given()
+                .get("/maven/io/quarkus/registry/quarkus-platforms/1.0-SNAPSHOT/quarkus-platforms-1.0-SNAPSHOT-all.json")
+                .then()
+                .statusCode(200)
+                 .contentType(ContentType.JSON)
```

Comment:
```suggestion
                .contentType(ContentType.JSON)
```

Minor formatting glitch